src/cloud-api-adaptor/test/provisioner/libvirt/provision_common.go

```diff
@@ -68,6 +69,9 @@ func NewLibvirtProvisioner(properties map[string]string) (pv.CloudProvisioner, e
 		vol_name = properties["libvirt_vol_name"]
 	}
 
+	// alternate_vol_name is used for multi-podvm testing.
+	alternate_vol_name := "another-podvm-base.qcow2"
```

Comment:
Is there a possibility to extract this as a constant, so we can reference it in libvirt_test rather than having to copy the value there?

Reply:
Actually I tried to do this way, but unfortunately `common.go` inside the `e2e` folder is not extending to this `provision_common.go`, also thought about adding it to `common.go` under `provisioner`, which is again not referenced on  `libvirt_test`/`common_suite` under e2e. if it sounds fair to import, i can try this out..

Reply:
Ah, I see. One approach we could take (which I'm not sure is even good) is create the const in `libvirt/provision_common.go` and then we can reference it in `libvirt_test.go` and pass it in as an argument to the common_test.go version? Otherwise I think we can leave it for now.

Reply:
Thanks Steve, this sounds like an idea, will test it out..